tokenserver-db/Cargo.toml

```diff
@@ -14,7 +14,6 @@ slog-scope.workspace = true
 thiserror.workspace = true
 
 async-trait = "0.1.88"
-# diesel 1.4
```

Comment:
nit: also got one of these in the syncstorage-db-common [Cargo.toml](https://github.com/mozilla-services/syncstorage-rs/blob/19b6176d5b9d7848b57954a489d494394c175798/syncstorage-db-common/Cargo.toml#L19)